### PR TITLE
Adding login template strings for translation

### PR DESCRIPTION
**Pull request by sg00dwin**
Related to: https://github.com/openshift/oauth-server/pull/71

### CODE REVIEW

File: frontend/public/locales/en/public.json [hunk 1]
```diff
@@ -728,5 +728,11 @@
...
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
-}
\ No newline at end of file
+  "Enabled": "Enabled",
+  "LogInToYourAccount": "Log in to your account",
+ 	"LogIn": "Log in",
+ 	"LoginTitle": "Login",
+ 	"WelcomeTo": "Welcome to {{platformTitle}}",
+ 	"LogInWith": "Log in with {{providerName}}",
```
Comment (sg00dwin) [inline frontend/public/locales/en/public.json:736]: @rebeccaalpert I included `{{platformTitle}}` and `{{providerName}}` to give the translators context. Is this acceptable?

Comment (rebeccaalpert) [reply frontend/public/locales/en/public.json]: Works for me! Thanks.

Comment (rebeccaalpert) [reply frontend/public/locales/en/public.json]: It's better if the phrase is the key on the left. The translators only receive the key value and they're not going to be able to translate LoginTitle, etc.

Comment (sg00dwin) [reply frontend/public/locales/en/public.json]: updated the keys
```
"Log in to your account": "Log in to your account",
"Log in": "Log in",
"Login": "Login",
"Welcome to {{platformTitle}}": "Welcome to {{platformTitle}}",
"Log in with {{providerName}}": "Log in with {{providerName}}",
"Error": "Error"
```

Comment (rebeccaalpert) [reply frontend/public/locales/en/public.json]: Thank you!

**Review by rebeccaalpert — APPROVED** [review]
Looks good to me. It just looks like you need to run `yarn i18n`. I think the script allows for manual entries. If that turns out to be incorrect and these are overwritten, you may need to add some parser comments somewhere instead like `// t('public~Log in')`.

File: frontend/public/locales/en/public.json [hunk 2]
```diff
@@ -728,5 +728,11 @@
...
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
-}
\ No newline at end of file
+  "Enabled": "Enabled",
+  "Log in to your account": "Log in to your account",
+  "Log in": "Log in",
+  "Login": "Login",
+  "Welcome to {{platformTitle}}": "Welcome to {{platformTitle}}",
+  "Log in with {{providerName}}": "Log in with {{providerName}}",
```
Comment (spadgett) [inline frontend/public/locales/en/public.json:736]: This won't necessarily work for the OAuth server login page. See https://github.com/openshift/oauth-server/pull/71#discussion_r609880583

**Review by spadgett — CHANGES_REQUESTED** [review]
@sg00dwin Don't manually add these messages to the public.json file. IIRC `yarn i18n` will remove them when run. We need to add i18next-parser comments to one our of source files like

```
// t('Log in')
// ...etc...
```

Then `yarn i18n` will automatically add them to public.json.

File: frontend/public/locales/en/public.json (hunk 2 above)
Comment (sg00dwin) [reply frontend/public/locales/en/public.json:736]: > @sg00dwin Don't manually add these messages to the public.json file. IIRC `yarn i18n` will remove them when run. We need to add i18next-parser comments to one our of source files like

@spadgett is there a recommended file to add these parser comments? Or should I just pick one?

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**